test/remoteCollection/collect.js

```diff
@@ -604,15 +610,47 @@ describe('test/remoteCollection/collect.js >', () => {
 
       const spy = sinon.spy(request, 'get');
       collect.prepareRemoteRequest(generator)
-      .then(() => {
-        expect(spy.returnValues[0]._proxyUri).to.be.equal(undefined);
-        spy.restore();
-        done();
-      })
-      .catch((err) => {
-        spy.restore();
-        done(err);
-      });
+        .then(() => {
```

Comment:
I see you are making efforts to correct (or change stylings in your PRs). Maybe its time we decide on a common guideline for format styling as a team?

Reply:
we have *both* .jscsrc and .eslintrc in the root of this repo - let's have this conversation outside of this PR so the PR doesn't get derailed

Reply:
I am just following the eslint convention over webstorm (is that wrong?)